sdks/java/core/src/main/java/org/apache/beam/sdk/testing/NeedsRunner.java

```diff
@@ -21,8 +21,23 @@
 import org.apache.beam.sdk.runners.PipelineRunner;
 
 /**
- * Category tag for validation tests which utilize {@link TestPipeline} for execution and expect
- * to be executed by a {@link PipelineRunner}.
+ * Category tag for tests which utilize {@link TestPipeline} for execution and
+ * {@link PAssert} for validation. Tests in this category require a
```

Comment:
It is not necessary that they use `PAssert`. It just means that "this test cannot run without a runner set up for `TestPipeline`".